backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSpecimenType.java

```diff
@@ -55,4 +60,12 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(deviceType, specimenType);
   }
+
+  public Device getFhirDevice() {
+    return convertToDevice(deviceType);
```

Comment:
Having these methods at the `DeviceSpecimenType` level feels a little squidgy to me, since theoretically all these `model` classes should be direct representations of the corresponding database tables and nothing more.
I don't have a useful suggestion on what else we could do and I know this practice is in place elsewhere too, just something to consider from an architecture perspective.